### ISSUE TITLE
Testing accuracy on CNN Sentiment analysis

### ISSUE THREAD

**Issue by Algogator** [bentrevett/pytorch-sentiment-analysis#51]
The validation accuracy is higher than the training accuracy?

**Comment by bentrevett** [comment]
For notebook 4? The validation and test are both lower than the training accuracy.
